edge-hub/watchdog/linux/Makefile

```diff
@@ -0,0 +1,14 @@
+TARGET=target/release
+CARGOFLAGS=--manifest-path=$(SRCDIR)/Cargo.toml
+CARGO=cargo
+SRCDIR?=.
+
+# TODO: should we have a make target that doesn't build release?
+all:
+	$(CARGO) build $(CARGOFLAGS)
```

Comment:
Should we remove this? It might be safer to only build release since the only place this makefile is used is in the build pipeline.